packages/block-editor/src/components/block-list/block.js

```diff
@@ -97,6 +99,9 @@ function BlockListBlock( {
 	toggleSelection,
 	onShiftSelection,
 	onSelectionStart,
+
```

Comment:
```suggestion
```

Reply:
haha it was intentional :P to separate animation related pros but I guess it's not consistent. I'll remove :)